.github/workflows/ci.yaml

```diff
@@ -0,0 +1,101 @@
+name: Publish to ECR
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "helm/**"
+      - "**.md"
+      - "**.txt"
+      - "Jenkinsfile"
+      - ".git**"
+
+  pull_request:
+    paths-ignore:
+      - "helm/**"
+      - "**.md"
+      - "**.txt"
+      - "Jenkinsfile"
+      - ".git**"
```

Comment:
This seems perhaps too defensive to me. Are we expecting problems if we have the occasional identical build? It feels unlikely we'd ever get an exhaustive list of things here and potentially a bit dangerous if applied uniformly(some apps may use a markdown or text file)

It also has the practical problem that we can't see if changes to the workflow script do anything.